docs/deploying.md

```diff
@@ -2,155 +2,135 @@
 +++
 title = "Deploying a registry server"
 description = "Explains how to deploy a registry server"
-keywords = ["registry, service, images,  repository"]
+keywords = ["registry, service, images, repository"]
 [menu.main]
 parent="smn_registry"
 weight=3
 +++
 <![end-metadata]-->
 
-
 # Deploying a registry server
 
-You obviously need to [install Docker](https://docs.docker.com/installation/) (remember you need **Docker version 1.6.0 or newer**).
+You need to [install Docker version 1.6.0 or newer](https://docs.docker.com/installation/).
 
-## Getting started
+## Running on localhost
 
 Start your registry:
 
-     $ docker run -d -p 5000:5000 \
-     		--restart=always --name registry registry:2
+    docker run -d -p 5000:5000 --restart=always --name registry registry:2
```

Comment:
The registry:2 tag will be updated to point to the latest version of registry 2.x, right?